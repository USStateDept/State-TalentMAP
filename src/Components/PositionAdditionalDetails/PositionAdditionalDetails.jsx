import React from 'react';
import PropTypes from 'prop-types';

const PositionAdditionalDetails = ({ content }) => (
  <div className="usa-grid position-details-additional">
    <div className="usa-width-one-whole">
      <p className="position-details-additional-body">
        <span className="title">About</span>
        <br />
        { content }
      </p>
    </div>
  </div>
);

PositionAdditionalDetails.propTypes = {
  content: PropTypes.string.isRequired,
};

export default PositionAdditionalDetails;
