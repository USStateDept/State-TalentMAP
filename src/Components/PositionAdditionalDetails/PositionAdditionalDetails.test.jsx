import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import PositionAdditionalDetails from './PositionAdditionalDetails';

describe('PositionAdditionalDetails', () => {
  it('is defined', () => {
    const wrapper = shallow(<PositionAdditionalDetails content="test" />);
    expect(wrapper).toBeDefined();
  });

  it('matches snapshot', () => {
    const wrapper = shallow(<PositionAdditionalDetails content="test" />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
