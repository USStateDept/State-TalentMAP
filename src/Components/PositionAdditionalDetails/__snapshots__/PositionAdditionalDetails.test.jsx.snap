// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PositionAdditionalDetails matches snapshot 1`] = `
<div
  className="usa-grid position-details-additional"
>
  <div
    className="usa-width-one-whole"
  >
    <p
      className="position-details-additional-body"
    >
      <span
        className="title"
      >
        About
      </span>
      <br />
      test
    </p>
  </div>
</div>
`;
